packages/kit/src/runtime/client/renderer.js

```diff
@@ -384,6 +384,16 @@ export class Renderer {
 					deep_linked.scrollIntoView();
 				} else {
 					scrollTo(0, 0);
+					if (document.body.scrollTop > 0) document.body.scrollTo(0, 0);
+					// In some cases it is needed scroll first root element of target that
```

Comment:
I don't understand this comment. Can you expand it to give an example of such a case?